OloEngine/src/OloEngine/Renderer/Font.cpp

```
@@ -1,140 +1,160 @@
-        std::string fileString = filepath.string();
+        const auto fileSize = file.tellg();
+        if (fileSize <= 0)
+        {
+            OLO_CORE_ERROR("Failed to determine font file size (tellg={}) for: {}", static_cast<std::streamoff>(fileSize), m_Path);
+            return;
+        }
 
-        // TODO(olbu): msdfgen::loadFontData loads from memory buffer which we'll need
-        msdfgen::FontHandle* font = msdfgen::loadFont(ft, fileString.c_str());
-        if (!font)
+        constexpr std::streamoff kMaxFontFileSize = 64 * 1024 * 1024; // 64 MB sanity cap
```

Clang (14.0.6)

[warning] variable 'kMaxFontFileSize' is not initialized

(cppcoreguidelines-init-variables)